inference/core/env.py

```diff
@@ -260,6 +260,10 @@
 # Model ID, default is None
 MODEL_ID = os.getenv("MODEL_ID")
 
+# Enable the builder, default is False
+ENABLE_BUILDER = str2bool(os.getenv("ENABLE_BUILDER", False))
```

Comment:
seems fine, maybe `ENABLE_WORKFLOW_BUILDER_EMBED` `ENABLE_WORKFLOW_BUILDER_UI` is more specific?  thinking in overall inference context `builder` could mean a lot of things